HPXMLtoOpenStudio/tests/test_defaults.rb

```diff
@@ -4449,19 +4586,24 @@ def _test_default_furniture_mass_values(furniture_mass, area_fraction, type)
     assert_equal(type, furniture_mass.type)
   end
 
-  def _test_default_central_air_conditioner_values(cooling_system, shr, compressor_type, fan_watts_per_cfm, charge_defect_ratio,
+  def _test_default_central_air_conditioner_values(cooling_system, shr, compressor_type, fan_watts_per_cfm, cooling_airflow_cfm, charge_defect_ratio,
                                                    airflow_defect_ratio, cooling_capacity, cooling_efficiency_seer, crankcase_heater_watts, cooling_autosizing_factor)
     assert_equal(shr, cooling_system.cooling_shr)
     assert_equal(compressor_type, cooling_system.compressor_type)
     assert_equal(fan_watts_per_cfm, cooling_system.fan_watts_per_cfm)
+    if cooling_airflow_cfm.nil? # nil implies an autosized value
+      assert(cooling_system.cooling_airflow_cfm > 0)
+    else
+      assert_in_delta(cooling_airflow_cfm, cooling_system.cooling_airflow_cfm, 1.0)
+    end
```

Comment:
Leaving these even though we always pass nil into them; we may use them in the future.

Reply:
This seems fine.